containers/compose-base.yaml

```diff
@@ -105,12 +105,12 @@ services:
       - ../.env
     ports:
       - ${POSTGRES_PORT}:${POSTGRES_PORT}
-    user: "${HOST_UID:-1000}:${HOST_GID:-1000}"
```

Comment:
any reason for removing this line?

Reply:
postgres user has been properly configured with correct ID and GUI and the container initiates with this user [here](https://github.com/thegraphnetwork/EpiGraphHub/pull/189/files#diff-3fbf821626d0c9d9f6942f5817062a5db43be6434bd83f6906d0492b321f490fR37)

Reply:
oh, that is right .. sorry .. I missed that it was about the postgres service XD
good catch!